server/store.go

```diff
@@ -26,7 +26,7 @@ type ResourceKind int
 
 const (
 	leaderKind ResourceKind = iota + 1
-	storageKind
+	regionKind
```

Comment:
I think `peerKind` would be better, since the 2 kinds resources are _region's leader_ and _region's peer_.